components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3760,7 +3763,7 @@ function doValidation() {
             <td>
                 <div class="sectionCheckbox">
                     <input id="isEnableAssertionSigning" name="isEnableAssertionSigning"
-                           type="checkbox" <%=enableAssertionSigningChecked%>/>
+                           type="checkbox" <%=Encode.forHtmlAttribute(enableAssertionSigningChecked)%>/>
```

Comment:
The variable value is hardcoded. Further this is not an attribute. The variable fully represents the attribute key and value here. So no need to encode.